bundles/org.openhab.binding.revogismartstripcontrol/src/main/java/org/openhab/binding/revogismartstripcontrol/internal/api/DiscoveryRawResponse.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.revogismartstripcontrol.internal.api;
+
+import java.util.Objects;
+
+/**
+ * @author Andi Bräu - Initial contribution
+ */
+public class DiscoveryRawResponse {
```

Comment:
All classes need to have `@NonNullByDefault`

Reply:
The docs say `Classes (except data transfer objects (DTO)) must be annotated with @NonNullByDefault`. This class is a data class. All non-data classes are annotated with NonNullByDefault.

Reply:
The docs say `Classes (except data transfer objects (DTO)) must be annotated with @NonNullByDefault`. This class is a data class. All non-data classes are annotated with NonNullByDefault

Reply:
Leave as is then.